cep-plugin-infrastructure.md

```diff
@@ -0,0 +1,103 @@
+<table>
+<tr><td> Title </td><td> Implement initial conda plugin mechanism </td>
+<tr><td> Status </td><td> Draft  </td></tr>
+<tr><td> Author(s) </td><td> Bianca Henderson, Jannis Leidel </td></tr>
+<tr><td> Created </td><td> July 5, 2022 </td></tr>
+<tr><td> Updated </td><td> July 6, 2022 </td></tr>
+<tr><td> Discussion </td><td>https://github.com/conda-incubator/ceps/pull/32</td></tr>
+<tr><td> Implementation </td><td>https://github.com/conda/conda/pull/11435</td></tr>
+</table>
+
+## Abstract
+
+In order to enable customization and extra features that are compatible with and discoverable by `conda` (but do not necessarily ship as a default part of the `conda` codebase), we would like to implement an official `conda` plugin mechanism.
```

Comment:
I think this should refer to [`CEP 2`](https://github.com/conda-incubator/ceps/blob/main/cep-2.md) so we start a chain of CEPs referring to plugin-specific CEPs.